cli/src/test/java/com/devonfw/tools/ide/io/FileAccessImplTest.java

```diff
@@ -344,7 +344,8 @@ private void assertSymlinksExist(Path dir) {
    * Checks if the symlinks are broken. This is used by the tests of {@link FileAccessImpl#symlink(Path, Path, boolean)}.
    *
    * @param dir the {@link Path} to the directory where the symlinks are expected.
-   * @param readLinks - {@code true} if the symbolic link shall be read with {@link Files#readSymbolicLink(Path)}, this does not work for Windows junctions.
+   * @param readLinks - {@code true} if the symbolic link shall be read with {@link Files#readSymbolicLink(Path)}, this does not work for Windows
+   *     junctions.
```

Comment:
This still looks like diff-war to me.
Can we clarify why this is still happening?
Are we now all working on `main` for our settings and is everybody up-to-date (`ide update`)?

Reply:
I am working on an updated `main`.